### DISCUSSION TITLE
Issues creating text styles from typography tokens

### DISCUSSION THREAD

**Discussion by chriskerr-performio** [six7/figma-tokens#631] [General]
Hey @six7!

I'm having some issues with applying typography tokens as styles.

My typography tokens are set up to reference via aliases the related tokens per the screengrab attached.

However when I use Create Styles, none of the actual values are applied to the style. All the styles are created, but they all default to 12/Auto and the wrong font, weight, etc.

I note that in the Edit Token dialog for typography, when you enter an alias, it does not return the value of the alias (unlike say color, where is tells you the hex value and shows the colour preview swatch.

Is this a bug, or am I doing it wrong?

I also note that by default textDecoration and textCase use the older "$" notation instead of the curly brackets for the alias. This was auto-populated when i imported my styles into Figma Tokens. Is this intentional and should I manually change these to curlys?

<img width="240" alt="Screen Shot 2022-04-22 at 05 01 19" src="https://user-images.githubusercontent.com/100885654/164534245-2b757b75-d36f-44b7-8971-07d99c87e336.png">

<img width="750" alt="Screen Shot 2022-04-22 at 04 18 10" src="https://user-images.githubusercontent.com/100885654/164527401-846ac9cc-69b6-4adc-bf06-11bac5084c7d.png">

**Comment by six7** [comment]
Is the font weight / family combination available in Figma? I see that you used font weight 500. But is that what the weight is called in Figma?

**Reply by Thomas-Neulabs** [reply]
have you solved this issue? I have the same problem and I cannot figure out how to solve it.
thanks

**Comment by yumyo** [comment]
I have just run into this issue: 

I have created tokens by importing styles => 
exported the token to a file => 
created a new document => 
imported the tokens from the file created before (also tried copy/past the JSON) => 
created styles from tokens in the new document = all typography styles (comp) are 12/auto

**Reply by six7** [reply]
I'd need to see a json to understand what's going on there

**Reply by yumyo** [reply]
Hey @six7, this is what I am using [https://gist.github.com/yumyo/c9d...](https://gist.github.com/yumyo/c9da6b29652753dc21e2b31acb4aeda1) 
* parsed with  "token-transformer"
